igvc_gazebo/nodes/scan_to_pointcloud/scan_to_pointcloud.cpp

```diff
@@ -8,11 +9,20 @@
 #include <sensor_msgs/LaserScan.h>
 #include <tf/transform_datatypes.h>
```

Comment:
Make sure not to include the same headers twice in the source and header files. Your includes should be in your header file.